flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -582,16 +582,41 @@ public void notifySlotAvailable(
 	}
 
 	@Override
-	public CompletableFuture<TransientBlobKey> requestTaskManagerFileUpload(ResourceID taskManagerId, FileType fileType, Time timeout) {
-		log.debug("Request file {} upload from TaskExecutor {}.", fileType, taskManagerId);
+	public CompletableFuture<TransientBlobKey> requestTaskManagerFileUploadByType(ResourceID taskManagerId, FileType fileType, Time timeout) {
+		log.debug("Request file which type is {}  upload from TaskExecutor {}.", fileType, taskManagerId);
```

Comment:
Unfortunately this log message and the previous one is not valid English. My suggestion is:
```
log.debug("Request {} file upload from TaskExecutor {}.", fileType, taskManagerId);
```